test/specs/modules/Sticky/Sticky-test.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react'
+import Sticky from 'src/modules/Sticky/Sticky'
+import { sandbox } from 'test/utils'
+import * as common from 'test/specs/commonTests'
+
+// Scroll to the top of the screen
+const scrollToTop = (wrapper, contextEl, { bottomOffset, offset, height }) => {
+  const instance = wrapper.instance()
+  instance.triggerRef = { getBoundingClientRect: () => ({ top: offset }) }
+  instance.stickyRef = { getBoundingClientRect: () => ({ height, top: offset }) }
+  contextEl.getBoundingClientRect = () => ({ bottom: height + offset + bottomOffset })
+  window.dispatchEvent(new Event('scroll'))
+}
+
+// Scroll until the trigger is not visible
+const scrollAfterTrigger = (wrapper, contextEl, { bottomOffset, offset, height }) => {
+  const instance = wrapper.instance()
+  instance.triggerRef = { getBoundingClientRect: () => ({ top: offset - 1 }) }
+  instance.stickyRef = { getBoundingClientRect: () => ({ height }) }
+  contextEl.getBoundingClientRect = () => ({ bottom: window.innerHeight - bottomOffset + 1 })
+  window.dispatchEvent(new Event('scroll'))
+}
+
+// Scroll until the context bottom is not visible
+const scrollAfterContext = (wrapper, contextEl, { offset, height }) => {
+  const instance = wrapper.instance()
+  instance.triggerRef = { getBoundingClientRect: () => ({ top: offset - 1 }) }
+  instance.stickyRef = { getBoundingClientRect: () => ({ height }) }
+  contextEl.getBoundingClientRect = () => ({ bottom: -1 })
+  window.dispatchEvent(new Event('scroll'))
+}
+
+// Scroll to the last part of the context
+const scrollToContextBottom = (wrapper, contextEl, { offset, height }) => {
+  const instance = wrapper.instance()
+  instance.triggerRef = { getBoundingClientRect: () => ({ top: offset - 1 }) }
+  instance.stickyRef = { getBoundingClientRect: () => ({ height }) }
+  contextEl.getBoundingClientRect = () => ({ bottom: height + 1 })
+  window.dispatchEvent(new Event('scroll'))
+}
+
+describe('Sticky', () => {
+  common.isConformant(Sticky)
+  common.rendersChildren(Sticky)
+
+  it('should create two divs', () => {
+    mount(<Sticky />).html().should.be.equal('<div><div></div><div></div></div>')
```

Comment:
I think we can use `shallow` there instead of `mount`, also it will be more clear if we will use another enzyme's methods.

```js
const children = shallow(<Sticky />).children()

children.should.have.length(2)
children.everyWhere(child => child.should.have.tagName('div'))

```